src/lib/vaccinations_utils.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pandas import DataFrame
+
+
+def add_total_persons_vaccinated_estimate(data: DataFrame):
```

Comment:
Can you make a copy instead of editing the data inplace?

Reply:
thanks for reviewing!

I've done it slightly differently..the function now returns a series and no longer edits the data inplace

so now its upto the caller. in the two callers I still edit inlace like below
data["total_persons_vaccinated"] = estimate_total_persons_vaccinated(data)

that should be OK as there are other inlace edits in the datasources 

WDYT?